keyservice/keyservice.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
+
+message Key {
+	oneof key_type {
+		KmsKey kms_key = 1;
+		GpgKey gpg_key = 2;
```

Comment:
nit: gpg is an implementation of pgp. I prefer to use the term `pgp` everywhere, not `gpg`. (I know, it's confusing)

Reply:
I've replaced all usages of GPG to PGP, except when it specifically refers to the GPG binary